test/e2e/multicast_test.go

```diff
@@ -141,7 +150,118 @@ func TestMulticast(t *testing.T) {
 		if !multipleInterfacesFound {
 			t.Skip("Skipping test because none of the Nodes has more than one multicast enabled interface")
 		}
-		runTestMulticastForwardToMultipleInterfaces(t, data, nodeIdx, 3464, "224.3.4.13", nodeMulticastInterfaces[nodeIdx])
+		testMulticastForwardToMultipleInterfaces(t, data, nodeIdx, 3464, "224.3.4.13", nodeMulticastInterfaces[nodeIdx])
+	})
+	t.Run("testMulticaststats", func(t *testing.T) {
+		skipIfNumNodesLessThan(t, 2)
+		testcases := []multicastStatsTestcase{
+			{
+				name: "testMulticastNetworkPolicyStats",
+				senderConfigs: []senderConfigs{
+					{nodeName: nodeName(0), name: "test2-sender-1", IPs: []string{"225.20.2.2", "225.20.2.3"}, sendSessions: 10},
```

Comment:
nit: the sequence in the name is wrong: test2->test3->test

Reply:
updated